lib/contract.ts

```diff
@@ -0,0 +1,57 @@
+import ethers, { providers, Contract, Wallet } from "ethers";
+import PODcontracts from "./POD-contracts.json";
+
+type ClaimProps = {
+  account: string;
+  code: string;
```

Comment:
This will also need to support passing multiple nft contracts addresses or names so we init the red pil, green pil or spork and sour based on what the user is trying to claim.

Reply:
added podContract var

Reply:
done